components/dashboard/src/components/UsageView.tsx

```diff
@@ -160,13 +175,67 @@ function UsageView({ attributionId }: UsageViewProps) {
 
     const headerTitle = attributionId.kind === "team" ? "Team Usage" : "Personal Usage";
 
+    const DateDisplay = forwardRef((arg: any, ref: any) => (
+        <div
+            className="px-2 p-1 text-gray-500 bg-gray-100 dark:text-gray-400 dark:bg-gray-800 rounded-md cursor-pointer flex items-center"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            <div className="font-medium">{arg.value}</div>
+            <div>
+                <svg
+                    width="20"
+                    height="20"
+                    fill="currentColor"
+                    xmlns="http://www.w3.org/2000/svg"
+                    onClick={arg.onClick}
+                    ref={ref}
+                >
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M5.293 7.293a1 1 0 0 1 1.414 0L10 10.586l3.293-3.293a1 1 0 1 1 1.414 1.414l-4 4a1 1 0 0 1-1.414 0l-4-4a1 1 0 0 1 0-1.414Z"
+                    />
+                    <title>Change Date</title>
+                </svg>
+            </div>
+        </div>
+    ));
+
     return (
         <>
             <Header
-                title={headerTitle}
-                subtitle={`${new Date(startDateOfBillMonth).toLocaleDateString()} - ${new Date(
-                    endDateOfBillMonth,
-                ).toLocaleDateString()} (updated every 15 minutes).`}
+                title={
+                    <div className="flex items-baseline">
+                        <h1 className="tracking-tight">{headerTitle}</h1>
+                    </div>
+                }
+                subtitle={
+                    <div className="tracking-wide flex mt-3 items-center">
```

Comment:
**issue(non-blocking)**: This structure breaks slightly the heigh of the header but let's fix this in https://github.com/gitpod-io/gitpod/issues/6764 unless you'd like to move this element in the `div` above, see subtitle and bottom border.

![Frame 1136](https://user-images.githubusercontent.com/120486/201177871-ec0a0008-789e-48c0-b9ee-802c68a457f1.png)